enhancements/scheduling/prevent-user-workloads-from-being-scheduled-on-control-plane-nodes.md

```diff
@@ -229,60 +249,133 @@ spec:
         operator: DoesNotExist
 ```
 
-4. Namespace Labeling for Exceptions:
+4. Service Account to be used by the workload:
 
-- Namespace Administrators are tasked with applying specific labels to their namespaces if exemptions to the default scheduling policies are warranted. This procedure enables selected workloads within these namespaces to bypass the standard restrictions by tolerating the designated NoExecute taints (should be performed on control plane owned namespaces for control plane protection - should be performed on special node group namespaces if that is what needs to be protected).
+- Namespace Administrators or Application Developers need to have a valid Service Account to be used by the workload before the workload is created so the ValidatingAdmisisonPolicy can use the secondary authorization approach above checking if this Service Account can tolerate the relevant taints.
 
 ```
-oc label <NAMESPACE_NAME> openshift.io/control-plane=true
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: special-service-account
```

Comment:
Are admis/devs expected to use this exact SA name?

Reply:
No, besides the default provided VAP, other steps are suggestions.